sdk/textanalytics/azure-ai-textanalytics/samples/README.md

```diff
@@ -61,6 +61,7 @@ what you can do with the Azure Text Analytics client library.
 
 |**Advanced Sample File Name**|**Description**|
 |----------------|-------------|
+|[sample_analyze_sentiment_with_aspects.py][sample_analyze_sentiment_with_aspects] and [sample_analyze_sentiment_with_aspects_async.py][sample_analyze_sentiment_with_aspects_async]|Analyze sentiment in documents with granular analysis into individual aspects present in a sentence|
```

Comment:
I wonder if we need to include some note that this sample is only supported in 3.1

Reply:
good point, will add